src/components/IconButton/docs/examples.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import CodeExample from 'wix-storybook-utils/CodeExample';
+import * as ExtendedRawSource from '!raw-loader!./IconButtonExtendedExample.tsx';
+import * as ExtendedCSSRawSource from '!raw-loader!./IconButtonExtendedExample.st.css';
+import { MockSettings } from '../../../../stories/helperComponents/MockSettings';
+import { IconButtonExtendedExample } from './IconButtonExtendedExample';
+
+export const Example = () => (
```

Comment:
Instead of using the example with the `MockSetting` here, please look at the `Card`'s storybook `index` file.
we added a new function called `settingsPanel` that gives a new tab for this.
And instead have a simple example in this file here, maybe just two icons with the `disabled` `true` and `false`.